app/assets/javascripts/controllers/middleware_deployment/middleware_deployment_controller.js

```diff
@@ -0,0 +1,38 @@
+ManageIQ.angular.app.controller('middlewareAddDeploymentController', MiddlewareDeploymentCtrl);
```

Comment:
I'm wondering whether a proper location for this file is under middleware_deployment folder or middleware_server